frontend/appflowy_flutter/packages/flowy_infra_ui/lib/widget/separated_flex.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+
+typedef SeparatorBuilder = Widget Function();
+
+class SeparatedColumn extends Column {
+  final SeparatorBuilder separatorBuilder;
+
+  SeparatedColumn({
+    Key? key,
+    MainAxisAlignment? mainAxisAlignment,
+    CrossAxisAlignment? crossAxisAlignment,
+    MainAxisSize? mainAxisSize,
+    TextBaseline? textBaseline,
+    TextDirection? textDirection,
+    VerticalDirection? verticalDirection,
+    List<Widget> children = const <Widget>[],
+    required this.separatorBuilder,
+  }) : super(
+          key: key,
+          children: _insertSeparators(children, separatorBuilder),
+          mainAxisAlignment: mainAxisAlignment ?? MainAxisAlignment.start,
+          crossAxisAlignment: crossAxisAlignment ?? CrossAxisAlignment.center,
+          mainAxisSize: MainAxisSize.max,
+          textBaseline: textBaseline,
+          textDirection: textDirection,
+          verticalDirection: verticalDirection ?? VerticalDirection.down,
+        );
```

Comment:
Use super constructor
```suggestion
  SeparatedColumn({
    super.key,
    super.mainAxisAlignment = MainAxisAlignment.start,
    super.crossAxisAlignment =  CrossAxisAlignment.center,
    super.mainAxisSize = MainAxisSize.max,
    super.textBaseline,
    super.textDirection,
    super.verticalDirection = VerticalDirection.down,
    List<Widget> children = const <Widget>[],
    required this.separatorBuilder,
  }) : super(
          children: _insertSeparators(children, separatorBuilder),
        );
```